src/main/java/uk/gov/justice/hmpps/prison/repository/jpa/model/OffenceIndicator.java

```diff
@@ -0,0 +1,32 @@
+package uk.gov.justice.hmpps.prison.repository.jpa.model;
+
+import lombok.*;
+
+import javax.persistence.*;
+import java.io.Serializable;
+
+@AllArgsConstructor
+@Builder
+@Data
+@EqualsAndHashCode(of = {"offence", "indicatorCode"}, callSuper = false)
+@NoArgsConstructor
+@Entity
+@Table(name = "OFFENCE_INDICATORS")
+@ToString
```

Comment:
Might not want to include `offence` in ToString - causes more loads

Reply:
Done